modules/ROOT/pages/instruction_set_architectures.adoc

```diff
@@ -0,0 +1,53 @@
+= Instruction Set Architectures
+
+An instruction set architecture, or ISA, is a specification detailing the design of instructions and operations on a processor.
+
+== Purpose
+ISAs are used to standardize operations between processors that might not share the same micro-architecture. They detail the set of instructions that a processor can execute, how those instructions are read and executed, and how the processor may communicate with devices.
```

Comment:
The sentence of the lead section and the second sentence of the "purpose" section are redundant, so merge them.
The first sentence of the "purpose" section implies that the primary purpose of ISA is standardization. However processors with their own unique, proprietary, undocumented ISA exist.